### PR TITLE
Improves usability of the Accounts Uplink Terminal

### PR DESCRIPTION
**Pull request by Contrabang**
## What Does This PR Do

When accounts are suspended, their background will show up as red. You can also see each person's balance from the menu now. Departmental accounts will also show when they're suspended now, and their symbol has been changed to a wallet instead of a person.

## Why It's Good For The Game
It improves the readability and usability of the terminal.

## Images of changes

image
image

## Testing

Messed around with suspending and unsuspending accounts, same with credits.

## Changelog
:cl:
tweak: Accounts will now show their status and balance in the Accounts Uplink Terminal
/:cl:

### CODE REVIEW

File: tgui/packages/tgui/interfaces/AccountsUplinkTerminal.js [hunk 1]
```diff
@@ -153,17 +160,25 @@ const DepartmentAccountsList = (properties, context) => {
...
+              <TableCell>Account Balance</TableCell>
             </Table.Row>
             {department_accounts
               .map((account) => (
-                <Table.Row key={account.account_number}
+                <Table.Row
+                  key={account.account_number}
+                  className={
+                    'AccountsUplinkTerminal__listRow--' + account.suspended
+                  }
```
Comment (Sirryan2002) [inline tgui/packages/tgui/interfaces/AccountsUplinkTerminal.js:172]: could you format the classname param like the other ones above?

File: tgui/packages/tgui/interfaces/AccountsUplinkTerminal.js [hunk 2]
```diff
@@ -123,6 +126,9 @@ const AccountsRecordList = (properties, context) => {
               .map((account) => (
                 <Table.Row
                   key={account.account_number}
+                  className={
+                    'AccountsUplinkTerminal__listRow--' + account.suspended
+                  }
```
Comment (Sirryan2002) [inline tgui/packages/tgui/interfaces/AccountsUplinkTerminal.js:131]: same here

File: tgui/packages/tgui/interfaces/AccountsUplinkTerminal.js (hunk 1 above)
Comment (Contrabang) [reply tgui/packages/tgui/interfaces/AccountsUplinkTerminal.js:172]: discussed on discord, keeping it consistent with security records

File: tgui/packages/tgui/interfaces/AccountsUplinkTerminal.js (hunk 2 above)
Comment (Contrabang) [reply tgui/packages/tgui/interfaces/AccountsUplinkTerminal.js:131]: ditto

**Review by Sirryan2002 — APPROVED** [review]
code looks good

**Pull request merged**